### PR TITLE
[LC-576] Add CI for MainNet Sync

### PR DESCRIPTION
**Pull request by metalg0su**
# Goal
- Add CI to check citizen -> MainNet sync

### CODE REVIEW

**Review by windies21 — APPROVED** [review]

File: .github/workflows/citizen.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        else
+          exit 0
+        fi
+      env:
+        target_height: "0xf"
+
+    - name: Notify
+      uses: Ilshidur/action-slack@master
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_URL }}
```
Comment (yakkle) [inline .github/workflows/citizen.yml]: where is target slack channel?

**Review by yakkle — COMMENTED** [review]
why don't you add action-badge?

**Comment by metalg0su** [comment]
> why don't you add action-badge?

Added! Thank you for your idea.

File: .github/workflows/citizen.yml (hunk 1 above)
Comment (metalg0su) [reply .github/workflows/citizen.yml]: Slack Webhook URL contains target channel.
> https://api.slack.com/messaging/webhooks

**Review by yakkle — APPROVED** [review]

**Pull request merged**